File: ming/0908/Remove Duplicates from Sorted Array.java
public class Solution {
    public int removeDuplicates(int[] nums) {
        if (nums.length <= 1){
            return nums.length;
        }
        int cur = nums[0];
        int j = 0;
        for (int i = 1; i < nums.length; i++){
            if (cur != nums[i]){
                j++;
                cur = nums[i];
                nums[j] = nums[i];
            }
        }
        return j + 1;
        
    }
}

File: mu_wang/9_08/Remove Duplicates from Sorted Array .py
class Solution(object):
    def removeDuplicates(self, A):
        """
        :type nums: List[int]
        :rtype: int
        """
        i = 0
        j = 0
        if len(A) <1:
            return 0
        A[i]=A[j]
        j+=1
        while j < len(A):
            if A[j]!= A[i]:
                i+=1
                A[i] = A[j]
            j+=1
        return i+1

        